server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1685,9 +1685,7 @@ public void removeInUseLogs(Set<DfsLogger> candidates) {
   public void checkIfMinorCompactionNeededForLogs(List<DfsLogger> closedLogs) {
 
     // grab this outside of tablet lock.
-    @SuppressWarnings("deprecation")
-    Property prop = tableConfiguration.resolve(Property.TSERV_WAL_MAX_REFERENCED,
-        Property.TSERV_WALOG_MAX_REFERENCED, Property.TABLE_MINC_LOGS_MAX);
+    Property prop = tableConfiguration.resolve(Property.TSERV_WAL_MAX_REFERENCED);
     int maxLogs = tableConfiguration.getCount(prop);
```

Comment:
This is another where there's only one property to resolve now. Can just inline the prop variable and get rid of the resolve call.